### PR TITLE
SAAS-295 filters fragment

### PR DESCRIPTION
**Pull request by yurkovychv**
[SAAS-295](https://jira.percona.com/browse/SAAS-295)

### CODE REVIEW

**Review by lunaticusgreen — APPROVED** [review]
👍

**Review by puneet0191 — APPROVED** [review]

**Comment by puneet0191** [comment]
@yurkovychv please run nightly job on your branch

File: pmm-app/tests/QAN/steps/qanActions.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ module.exports = {
     assert.notEqual(
       countAfter,
       countBefore,
-      `After applying Filter value ${countBefore} should not be equal to ${countAfter}`,
+      `After applying a Filter, count of queries should not be equal to ${countAfter}`,
```
Comment (atymchuk) [inline pmm-app/tests/QAN/steps/qanActions.js:179]: IMHO, we went from a more explicit to a less explicit error message. Now you have to research what is not equal to `countAfter`.

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: pmm-app/tests/QAN/steps/qanActions.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: @atymchuk It was done on purpose actually. If the assertion fails it says: After applying Filter value 1 should not be equal to 1

Comment (atymchuk) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: lol

**Review by atymchuk — APPROVED** [review]

File: pmm-app/tests/QAN/steps/qanActions.js (hunk 1 above)
Comment (puneet0191) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: lol good one, we don't really need to print values, I realized it is already available as part of expected and actual from Assert, we could just make it "After applying a Filter, count of queries was expected to change"

Comment (atymchuk) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: then double LOL

Comment (yurkovychv) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: yeah, we have a lot of such places

Comment (atymchuk) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: I'm learning something new every day. Will I ever stop?

Comment (puneet0191) [reply pmm-app/tests/QAN/steps/qanActions.js:179]: if you stop learning, I think it would not be a good place to be in :stuck_out_tongue:

**Pull request merged**